src/components/PostPageDetail/index.js

```diff
@@ -582,6 +585,11 @@ const PostPageDetailIdComponent = (props) => {
             isBackButton={true}
             source={SOURCE_PDP}
             height={getHeightHeader()}
+            isFollow={item?.is_following_target}
+            onPressFollUnFoll={() => {
+              followUnfollow(item);
```

Comment:
ini ga di async aja bang @asaadam  biar `is_following` nya ga langsung berubah? eh atau udah di handle kalo misal BE nya reject?

Reply:
hoo ini terkait ux prinsip kita juga ya? kalo BE reject kembali ke state awal?

ini kayanya aku juga implement di discovery tapi di level komponen, dan iya memang cukup bloated :"